modules/backend/controllers/Auth.php

```diff
@@ -40,9 +40,8 @@ public function __construct()
             return $response;
         })->only('signin');
 
-		// Wrapper to run the code only for HTTPs Connection	
-		if(isset($_SERVER["HTTPS"]) && $_SERVER["HTTPS"] === "on")
-        {
+		// Only run on HTTPS connections
+		if(isset($_SERVER["HTTPS"]) && $_SERVER["HTTPS"] === "on") {
```

Comment:
@ayumihamsaki space after `if` :)

Reply:
Fixed.

(What tool are you using to check the typos?)

Reply:
Eyes 😄 Hopefully will add travis-ci job, very soon, to check style.

Reply:
haha